cli/src/providers/any/APIProxy.py

```diff
@@ -27,3 +27,6 @@ def get_ips_for_feature(self, component_key):
         result.sort()
 
         return result
+
+    def login(self, env=None):
+        pass
```

Comment:
Maybe it will be good to add some comment there and say why we have done this that way?